index.js

```diff
@@ -159,7 +168,7 @@ app.get("/:sheet", (req, res) => {
 
     doc.getRows(1, (err, rows) => {
       if (err) {
-        res.statusCode(500).send('Unable to fetch Data from the database');
+        res.status(500).send('Unable to fetch Data from the database');
```

Comment:
Why have you changed here?

Reply:
statusCode is not correct, I have updated the function call after consulting with @rakaar

Reply:
@mukul-mehta, @thealphadollar   I was not sure at the moment, sorry. Once see this please
https://stackoverflow.com/questions/46129307/difference-between-response-statuscode-and-response-status